src/appkit.cs

```diff
@@ -17986,6 +17986,42 @@ interface NSComboBoxDelegate {
 		void SelectionIsChanging (NSNotification notification);
 	}
 
+	[BaseType (typeof(NSObject))]
+	[Model]
+	[Protocol]
+	interface NSTokenFieldCellDelegate {		
+		[Export ("tokenFieldCell:completionsForSubstring:indexOfToken:indexOfSelectedItem:")]
+		string [] GetCompletionStrings (NSTokenFieldCell tokenFieldCell, string substring, nint tokenIndex, ref nint selectedIndex);
+		
+		[Export ("tokenFieldCell:shouldAddObjects:atIndex:")]
+		NSArray ShouldAddObjects (NSTokenFieldCell tokenFieldCell, NSArray tokens, nuint index);
```

Comment:
the `NSArray` _could_ be changed to `NSObject[]` (both the return type and the parameter)

`An array of tokens to be inserted in the receiver at index.` it's not clear if a better type (than NSObject) could be used...

Documentation does not mention anything like the nested array above... but better check headers too :)

https://developer.apple.com/documentation/appkit/nstokenfieldcelldelegate/1523823-tokenfieldcell?language=objc